### ISSUE TITLE
Update to Swift 3.0 compliance 

### ISSUE THREAD

**Issue by DevPravin** [shuhrat10/STRatingControl#7]
Please update the component  files to Swift-3 compliant.

**Comment by Gerharbo** [comment]
Hi,

See this pull request for Swift 3 support.
https://github.com/shuhrat10/STRatingControl/pull/9

**Comment by ghost** [comment]
@Gerharbo maybe you can help me with this im trying to use this programmatically without storyboard. Im trying to set the filled image and star rating. this is my code. Thanks n advance

```
 var rating:STRatingControl = {
        let v = STRatingControl()
        v.translatesAutoresizingMaskIntoConstraints = false
        v.backgroundColor = .red
        v.heightAnchor.constraint(equalToConstant: 40).isActive = true
        v.widthAnchor.constraint(equalToConstant: 100).isActive = true

        return v

    }()
```

**Comment by Gerharbo** [comment]
You van change the rating with the help of the 'rating' property of STRatingControl.

In your code you can do the following:

```
  rating.rating = 3
```

Is this wat you were looking for?

**Comment by shuhrat10** [comment]
Swift 3 support has been released (1.1.0). Please update your pods.
